operator/v1/types_klusterlet.go

```diff
@@ -210,6 +210,10 @@ type LocalSecretsConfig struct {
 	HubConnectionTimeoutSeconds int32 `json:"hubConnectionTimeoutSeconds,omitempty"`
 }
 
+type KubeConfigSecret struct {
+	Name string `json:"name"`
```

Comment:
comment is needed, is this required field?

Reply:
Update.